internal/cmd/beta/network-area/routes/create/create.go

```diff
@@ -0,0 +1,181 @@
+package create
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/goccy/go-yaml"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/client"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/iaas"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	organizationIdFlag = "organization-id"
+	networkAreaIdFlag  = "network-area-id"
+	prefixFlag         = "prefix"
+	nexthopFlag        = "next-hop"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	OrganizationId *string
+	NetworkAreaId  *string
+	Prefix         *string
+	Nexthop        *string
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Creates a static route in a STACKIT Network Area (SNA)",
+		Long: fmt.Sprintf("%s\n%s\n",
+			"Creates a static route in a STACKIT Network Area (SNA).",
+			"This command is currently asynchonous only due to limitations in the waiting functionality of the SDK. This will be updated in a future release.",
```

Comment:
Let's just try to confirm first with the team if the request is even async or not. Because the created routes already come in the API response, right?

Reply:
Also the same applies for the delete, right?